### PR TITLE
Add a link to the PDF doc

### PR DESCRIPTION
**Pull request by mbcrump**

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/aws-professional/services.md](https://github.com/mbcrump/architecture-center/blob/patch-2/docs/aws-professional/services.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/aws-professional/services?branch=pr-en-us-253) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C9%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201710091914332636-253%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=j21wVG3haiY9yPBF6GRN%2FMLXz8TnCiAmiuJ85%2BknJjs%3D&st=2017-10-09T19%3A11%3A29Z&se=2017-11-09T19%3A16%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/aws-professional/services.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ This article helps you understand how Microsoft Azure services compare to Amazon
 
 In the following tables, there are multiple Azure services listed for some AWS services. The Azure services are similar to one another, but depth and breadth of capabilities vary.
 
+[Downloada PDF of the Azure & AWS Cloud Service Map](https://aka.ms/awsazureguide)
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: "Download a" > "Download a"

Comment (mbcrump) [reply docs/aws-professional/services.md]: Should be "Download a".

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/aws-professional/services.md](https://github.com/mbcrump/architecture-center/blob/patch-2/docs/aws-professional/services.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/aws-professional/services?branch=pr-en-us-253) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C12%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201710121652254435-253%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vE06pws5TjQJE9COI%2BaDubUiv%2BVsydNskBojK0HpZ8I%3D&st=2017-10-12T16%3A48%3A25Z&se=2017-11-12T16%3A53%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**